voice.go

```diff
@@ -302,7 +302,15 @@ func (v *VoiceConnection) open() (err error) {
 	// Connect to VoiceConnection Websocket
 	vg := fmt.Sprintf("wss://%s", strings.TrimSuffix(v.endpoint, ":80"))
 	v.log(LogInformational, "connecting to voice endpoint %s", vg)
-	v.wsConn, _, err = websocket.DefaultDialer.Dial(vg, nil)
+	dialer := v.session.VoiceDialer
+	if dialer == nil {
+		if v.session.Dialer != nil {
+			dialer = v.session.Dialer
```

Comment:
Is there a good reason to default back to session.Dialer as well? It's not really documented above and seems like an easy thing to set in the application:

```
someDialer := Dialer{}
session.Dialer = someDialer
session.VoiceDialer = someDialer
```

Reply:
I left it in because it made sense in my eyes.
Specifying nothing will fallback to the default dialer. So wouldn't it make more sense to fall back to the global dialer for voice comms if one is specified.
Makes more sense to me this way